README.md

```diff
@@ -1,2 +1,238 @@
+
+```python
+unmock_options = unmock.init(save=True)
+```
+You can also specify a specific location to save the directory:
+```python
+unmock_options = unmock.init(save=True, path=".")  # Saves in current path
+```
+Unmock refers to every mock by a unique hash. Individual mocks or groups
+of mocks can be saved by setting save to either a single hash or an
+array of hashes like so:
+
+```python
+unmock_options = unmock.init(save=["ahash", "anotherhash", "yetanotherhash"])
+```
+
+### Ignoring aspects of a mock
+
+Sometimes, you would like for two mocks of slightly API calls to be
+treated as equivalent by unmock. For example, you may want all `GET`
+calls to the same path with different headers to be served the same
+mock. To do this, use the `ignore` field of the unmock options object.
+You can do this while initializing unmock or afterwards (as shown before
+with ignoring `"story"`):
+
+```python
+# Option A:
+unmock_options = unmock.init()
+unmock_options.ignore("headers", "story")
+# Option B:
+unmock.init(unmock.UnmockOptions(ignore=["headers", "story"]))
+```
+
+The following fields may be ignored:
+
+* `headers`: the headers of the request
+* `hostname`: the hostname of the request
+* `method`: the method of the request (ie GET, POST, PUT, DELETE).
+Note that this is *case insensitive*!
+* `path`: the path of the request
+* `story`: the story of the request, meaning its order in a series of requests
+
+Ignore evaluates regular expressions, so you can also pass
+`"headers|path"` instead of `["headers", "path"]`. Furthermore,
+to ignore nested headers, pass an object such as
+`{headers: "Authorization" }`, or to match against the value of
+a header, `{headers: { Authorization: "Bearer *" }}`.
+
+### Adding a signature
+
+Sometimes, it is useful to sign a mock with a unique signature. This is
+useful, for example, when AB testing code that should serve two
+different mocks for the same endpoint in otherwise similar conditions.
+To do this, use the `signature` field of the unmock options object:
+
+```python
+# Option A:
+unmock_options = unmock.init()
+unmock_options.signature = "signature-for-this-particular-test"
+# Option B
+unmock.init(unmock.UnmockOptions(signature="signature-for-this-particular-test"))
+```
+
+### Whitelisting API
+
+If you do not want a particular API to be mocked, whitelist it.
+
+```python
+# Option A:
+unmock_options = unmock.init()
+unmock_options.whitelist = ["api.hubspot.com", "api.typeform.com"]
+# Option B:
+unmock.init(unmock.UnmockOptions(whitelist=["api.hubspot.com", "api.typeform.com"]))
+```
+
+### unmock.io tokens
+
+If you are subscribed to the [unmock.io](https://www.unmock.io) service,
+you can pass your unmock token directly to the unmock object.
+
+```
+unmock.init(token="my-token")
+```
+
+```js
+await unmock({
+  // ...
+  token: "my-token",
+  // ...
+}));
+```
+
+At a certain point this becomes a bit tedious, (even if very readable),
+at which point you will want to create a credentials file. See
```

Comment:
I'd assume every CI environment would use environment variable instead?

Reply:
Hm, yeah of course you can replace that with an environment variable.
I think the point was that in every test file you'd end up writing the `token=process.env.UNMOCK_TOKEN` or similar (most of this is copied straight from the `unmock-js` readme, so I'm unsure as well :smile: )